DataFormats/FTLRecHit/src/classes_def.xml

```diff
@@ -27,13 +30,6 @@
   <class name="edm::Wrapper<std::vector<edm::DetSet<FTLRecHit> > >"/>
   <class name="edm::Wrapper<edm::DetSetVector<FTLRecHit> >"/>
   <class name="edm::Wrapper<std::vector<std::vector<edm::DetSet<FTLRecHit> > > >"/>
-
-
-  <class name="FTLTrackingRecHitFromHit" ClassVersion="3">
```

Comment:
shouldn't this be dropped only with removal of  DataFormats/FTLRecHit/interface/FTLTrackingRecHit.h and DataFormats/FTLRecHit/src/FTLTrackingRecHit.cc files and corresponding lines in  DataFormats/FTLRecHit/src/classes.h?
Please restore the dictionary def or, alternatively, consistently remove these classes/instances as in #25063

Reply:
I've restored the dictionary.